assets/all.settings.php

```diff
@@ -47,6 +47,13 @@
     CURLOPT_PROXY_SSL_VERIFYHOST => 0,
     CURLOPT_PROXY_SSL_VERIFYPEER => FALSE,
   ];
+  // Specify non-HTTP versions of endpoints. This is required to make Cypress
+  // mocking work. It does not support ignoring self-signed certificates from
+  // Wiremock.
+  $config['openid_connect.settings.adgangsplatformen']['settings']['authorization_endpoint'] = 'http://login.bib.dk/oauth/authorize';
```

Comment:
This will also have effect for the production sites?
Is it not possible to make a check if we are running in wiremock/cypress context?

Reply:
This is wrapped in a `if (getenv('CI')) {` check on L42 to ensure that it does not affect production sites. It is the same approach we have been using for the Behat tests.